lib/run-compilation.js

```diff
@@ -26,19 +27,12 @@ module.exports = function runCompilation (options, compiler, done) {
         errors = results.filter(fileHasErrors);
       }
 
-      if (options.quiet === false) {
-        console.warn(options.formatter(results));
-      }
-
-      if (options.failOnError && errors.length) {
-        done(new Error(errorMessage));
-      } else {
-        done();
-      }
-    })
-    .catch(done);
+      if (options.quiet === false) console.warn(options.formatter(results));
+      if (options.failOnError && errors.length) done(new Error(errorMessage));
+      else done();
```

Comment:
I'm sorry to nitpick, but I am not a fan of this style. I think it's much clearer and more maintainable to use braces consistently. I'm surprised that the linter didn't catch this, but for now please do revert this. Thank you for sharing your opinion, however :)